test/unit/rules/no-link-to-positional-params-test.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'no-link-to-positional-params',
+
+  config: true,
+
+  good: [
+    '<LinkTo @route="about">About Us</LinkTo>',
+    '<LinkTo @route="post" @model={{@post}}>Read {{@post.title}}...</LinkTo>',
+    `<LinkTo @route="post.comment" @models={{array post comment}}>
+        Comment by {{comment.author.name}} on {{comment.date}}
+      </LinkTo>`,
+    `<LinkTo @route="posts" @query={{hash direction="desc" showArchived=false}}>
+        Recent Posts
+      </LinkTo>`,
+  ],
```

Comment:
Need some good examples using `{{link-to`

Reply:
Done